src/webchat/store/rating/rating-reducer.ts

```diff
@@ -0,0 +1,90 @@
+import { Reducer } from "redux";
+
+export interface RatingState {
+	ratingGiven: boolean;
```

Comment:
the boolean values can have more descriptive names. ratingGiven could also be "the rating that was given" and doesn't suggest that we're remembering whether a rating was given

Reply:
fixed